public/locales/en/peers.json

```
@@ -14,6 +15,7 @@
   "insertPeerAddress": "Insert the peer address you want to connect to.",
   "add": "Add",
   "example": "Example:",
+  "plusPeers": "Plus {number} more peers", 
```

Would this look better or worse? (feel free to ignore)

```suggestion
  "plusPeers": "+ {number} more peers", 
```

---

Don't really have a preference, just went with @jessicaschilling suggestion at the time 😄

---

@lidel - I was waffling between "plus" and "+" in the first place, so if you'd prefer "+", let's go with it 😊